.github/workflows/typescript-build.yml

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2023 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Check Typescript build
+
+permissions:
+  contents: read
+
+on:
+  push:
+    paths:
+      - '**.proto'
+      - 'gen/pb-typescript/**'
+  pull_request:
+    paths:
+      - '**.proto'
+      - 'gen/pb-typescript/**'
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        node-version:
+          - 14.17.0
+          - 14.x
+          - 16.13.0
+          - 16.x
+          - 18.0.0
+          - 18.x
+        platform:
```

Comment:
Do we need to run on all platforms? IMO just `ubuntu-latest` would be ok since we're *really* just checking that the release works IIUC.

Reply:
Yeah, good catch. I'd just lifted this out of the `sigstore-js` matrix without giving it much thought. Will pare it down.